main.rb

```diff
@@ -0,0 +1,159 @@
+# rubocop:disable Metrics\CyclomaticComplexity, Metrics/MethodLength
+require_relative 'book'
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    puts "Welcome to School Library App!\n\n"
+    command = nil
+    while command != '7'
+      commands
+
+      command = gets.chomp
+      case command
+      when '1'
+        list_books
+      when '2'
+        list_people
+      when '3'
+        create_person
```

Comment:
- [ ] It seems that option 3 is not working properly, I am not able to create a person successfully,  can you please fix this issue 😉